libraries/chain/system_calls.cpp

```diff
@@ -529,6 +529,15 @@ THUNK_DEFINE( void, apply_set_system_call_operation, ((const protocol::set_syste
 
    // Place the override in the database
    system_call::put_object( context, state::space::system_call_dispatch(), util::converter::as< std::string >( std::underlying_type_t< koinos::chain::system_call_id >( o.call_id() ) ), util::converter::as< std::string >( o.target() ) );
+
+   // Emit an event
+   protocol::set_system_call_event event;
+   event.set_call_id( o.call_id() );
+   event.mutable_target()->CopyFrom( o.target() );
+   std::string str;
+   event.SerializeToString( &str );
```

Comment:
Instead of having to declare `str` just for this you can use `event.SerializeAsString()`